### DISCUSSION TITLE
[Question]: Live View quality

### DISCUSSION THREAD

**Discussion by rdkartono** [blakeblackshear/frigate#12034] [Ask A Question] [answered]
### What is your question:

How can I set the config, so if Live View is showing multiple cameras, it used substream for network bandwidth conservation, but If I double click a camera become single view, it used main stream for high quality ? 

Is it possible ?

**Comment by NickM-27** [comment, marked as answer]
This is not currently possible

**Reply by rdkartono** [reply]
Thank you for the confirmation.
Really hope it can be implemented in future release .

**Comment by iscofield** [comment]
Is this still not possible?  I'd still like to be able to conserve bandwidth across my dozen cameras for the overview dashboard, but clicking on a camera get the detail/quality back, as opposed to having to drop everything down.

**Reply by NickM-27** [reply]
it will be possible in 0.16

**Reply by iscofield** [reply]
Awesome, thanks for the heads up!
